sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/KafkaCosmosExceptionsHelper.java

```diff
@@ -8,7 +8,7 @@
 import org.apache.kafka.connect.errors.ConnectException;
 import org.apache.kafka.connect.errors.RetriableException;
 
-public class CosmosExceptionsHelper {
+public class KafkaCosmosExceptionsHelper {
```

Comment:
Why is this renaming necessary - isn't kafka package sufficient?

Reply:
mostly just easier to differentiate between same class file between different projects. 
for example, `CosmosConstants -> KafkaCosmosContants. `

But no strong opinion, can revert back